config/config_test.go

```diff
@@ -297,6 +297,37 @@ func getOptionalParametersTestHelper(
 		}
 	})
 
+	t.Run(fmt.Sprintf("config that puts cartographer in offline mode and in localization mode %s", suffix), func(t *testing.T) {
+		cfgService := makeCfgService(imuIntegrationEnabled, cloudStoryEnabled)
+		cfgService.Attributes["existing_map"] = "test-file.pbstream"
+		cfgService.Attributes["data_rate_msec"] = 0
```

Comment:
why does the config have both data_rate_msec and camera.data_frequency_hz? shouldn't it have one or the other?

Reply:
`data_rate_msec` gets ignored if IMU is enabled, but I can only add it to the config if imu is enabled if you think that will make it clearer

Reply:
made this change!